### PR TITLE
fix: update settings pane validations

### LINKED ISSUE #18698: [Bug]: Duplicate page name are not allowed from Page manu but it is allowed from setting page
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Try to add same page name from Page manu and verify error is displayed as &Page& name is already being used 

image

Navigate to setting page and verify that the User is able to add duplicate page name 
image

### Steps To Reproduce

1. Click on add new page
2. Edit a Page name with already existing page name
3. Error is displayed that page name already exist
4. Navigate to page setting page and verify that user is able to add duplicate page name

### Public Sample App

_No response_

### Issue video log

_No response_

### Version

Cloud -release

### PR DESCRIPTION
**Pull request by eco-monk**
## Description

TL;DR -> Updated validation logic for fields in settings pane.

App name -  all characters are allowed now (shouldn’t be empty)
Page name - alphanumeric and accented characters are allowed (shouldn’t be empty), cannot use other entities name or any global keyword.
Custom slug - alphanumeric and "-” are allowed (can be empty)

Fixes #18698

Media
https://www.loom.com/share/64c488e2fc9f4a44802b10bfb3ccac84

## Type of change
- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

- Manual
- Cypress

### Dev activity
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

### QA activity:
- [ ] Test plan has been approved by relevant developers
- [ ] Test plan has been peer reviewed by QA
- [ ] Cypress test cases have been added and approved by either SDET or manual QA
- [ ] Organized project review call with relevant stakeholders after Round 1/2 of QA
- [ ] Added Test Plan Approved label after reveiwing all Cypress test

### CODE REVIEW

**Review by hetunandu — APPROVED** [review]

**Review by Aishwarya-U-R — COMMENTED** [review]
Cypress looks good!

**Review by akash-codemonk — APPROVED** [review]
